src/utils/multiformats/Cargo.toml

```diff
@@ -28,6 +28,6 @@ ed25519-dalek = { version = "2", default-features = false, features = ["std", "f
 hex = { version = "0.4", default-features = false, features = [] }
 rand = { version = "0.8", default-features = false, features = ["getrandom", "small_rng"] }
 sha3 = { version = "0.10", default-features = false, features = [] }
-serde = { version = "1", default-features = false, features = [] }
+serde = { version = "1", default-features = false, features = ["derive"] }
```

Comment:
Why `derive` feature is needed in this crate? it seems to have nothing to do with your changes.
I'd like our crates to depend on a minimal set of dependencies.

Reply:
Same for most other `Cargo.toml`s

Reply:
hey @sergiimk! thanks for review! removed.